pkg/modprovider/module.go

```diff
@@ -142,7 +142,7 @@ func (h *moduleHandler) Diff(
 	tf, err := h.prepSandbox(
 		ctx,
 		urn,
-		oldInputs,
+		newInputs,
```

Comment:
This looks suspect! Why the change.

Reply:
Ah. @Zaid-Ajaj explains that before the change oldInputs.DeepEquals(newInputs) made them the same.